static/js/addons/cl.utils.js

```diff
@@ -49,8 +49,12 @@ var Cl = window.Cl || {};
         },
 
         // INFO: simple mobile and tablet filter that can be extended
-        mobile: function () { return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i)); },
-        tablet: function () { return (navigator.userAgent.match(/iPad/i)); }
+        mobile: function () {
+            return (navigator.userAgent.match(/iPhone/i) || navigator.userAgent.match(/Android/i));
```

Comment:
@FinalAngel `$(window').width()` doesn't always return the same value that browser uses for determining current media query btw ;)

but yeah, the only time we need UA parsing is really edge cases on old mobile os versions, 99% of the time we wouldn't need that at all